app/routes/v1/systemSubscriptions.js

```diff
@@ -39,8 +41,77 @@ type: Opaque
   res.status( 200 ).send( razeeIdentitySecretYaml );
 };
 
+const getClusterSubscriptionSubscription = async(req, res) => {
+  let version;
+  let url;
+  if (RDD_STATIC_ARGS.length > 0) {
+    RDD_STATIC_ARGS.forEach(arg => {
+      if (arg.includes('clustersubscription')) {
+        version = arg.slice(arg.lastIndexOf('=') + 1);
+      }
+    });
+  }
+
+  if (version) {
+    url = `https://s3.us.cloud-object-storage.appdomain.cloud/razee-io/ClusterSubscription/${version}/us/resource.yaml`;
+  } else {
+    url = 'https://s3.us.cloud-object-storage.appdomain.cloud/razee-io/ClusterSubscription/latest/template/resource.yaml';
+  }
+
+  const clusterSubscriptionYaml = await request(url);
+
+  res.status( 200 ).send( clusterSubscriptionYaml );
+};
+
+const getRemoteResourceSubscription = async(req, res) => {
+  let version;
+  let url;
+  if (RDD_STATIC_ARGS.length > 0) {
+    RDD_STATIC_ARGS.forEach(arg => {
+      if (arg.includes('remoteresource')) {
+        version = arg.slice(arg.lastIndexOf('=') + 1);
+      }
+    });
+  }
+
+  if (version) {
+    url = `https://s3.us.cloud-object-storage.appdomain.cloud/razee-io/RemoteResource/${version}/us/resource.yaml`;
+  } else {
+    url = 'https://s3.us.cloud-object-storage.appdomain.cloud/razee-io/RemoteResource/latest/template/resource.yaml';
+  }
+
+  const remoteResourceYaml = await request(url);
+
+  res.status( 200 ).send( remoteResourceYaml );
+};
+
+const getWatchKeeperSubscription = async(req, res) => {
+  let version;
+  let url;
+  if (RDD_STATIC_ARGS.length > 0) {
+    RDD_STATIC_ARGS.forEach(arg => {
+      if (arg.includes('watch-keeper')) {
+        version = arg.slice(arg.lastIndexOf('=') + 1);
+      }
+    });
+  }
+
+  if (version) {
+    url = `https://s3.us.cloud-object-storage.appdomain.cloud/razee-io/WatchKeeper/${version}/us/resource.yaml`;
+  } else {
+    url = 'https://s3.us.cloud-object-storage.appdomain.cloud/razee-io/WatchKeeper/latest/template/resource.yaml';
+  }
+
+  const watchKeeperYaml = await request(url);
+
+  res.status( 200 ).send( watchKeeperYaml );
+};
+
 // /api/v2/systemSubscriptions/primaryOrgKey
 router.get('/primaryOrgKey', getOrg, asyncHandler(getPrimaryOrgKeySubscription));
+router.get('/clusterSubscription', asyncHandler(getClusterSubscriptionSubscription));
+router.get('/remoteResource', asyncHandler(getRemoteResourceSubscription));
+router.get('/watchKeeper', asyncHandler(getWatchKeeperSubscription));
```

Comment:
Would it be possible to have a single System Subscription to upload all the operators together?  I don't think we have a need to separately update cs, rr, wk.... but it may have some benefit I don't see yet?